libs/aws/langchain_aws/chat_models/bedrock.py

```diff
@@ -47,6 +47,42 @@ def convert_messages_to_prompt_llama(messages: List[BaseMessage]) -> str:
     )
 
 
+def _convert_one_message_to_text_llama3(message: BaseMessage) -> str:
```

Comment:
I propose one small change here, plus extra logic to support word in mouthing the assistant turn, for things like agent scratchpads etc living in the assistant turn,
```
def _convert_one_message_to_text_llama3(message: BaseMessage, lastMessage: bool) -> str:
    if isinstance(message, ChatMessage):
        message_text = f"<|start_header_id|>{message.role.capitalize()}<|end_header_id|>{message}<|eot_id|>"
    elif isinstance(message, HumanMessage):
        message_text = f"<|start_header_id|>user<|end_header_id|>{message.content}<|eot_id|>"
    elif isinstance(message, AIMessage):
        message_text = f"<|start_header_id|>assistant<|end_header_id|>{message.content}"
        if not lastMessage:
            message_text += "<|eot_id|>"
    elif isinstance(message, SystemMessage):
        message_text = f"<|start_header_id|>system<|end_header_id|>{message.content}"
    else:
        raise ValueError(f"Got unknown type {message}")
    return message_text


def convert_messages_to_prompt_llama3(messages: List[BaseMessage]) -> str:
    """Convert a list of messages to a prompt for llama3."""
    return "<|begin_of_text|>" + "".join(
        [_convert_one_message_to_text_llama3(message, i == len(messages) - 1)
         for i, message in enumerate(messages)]
    ) + "<|start_header_id|>assistant<|end_header_id|>"
```

This would allow stuff along these lines.
```
messages = [HumanMessage(content="list 5 colors"),AIMessage(content="No, I don't want to!")]
```

Reply:
I am not totally sure I understand the use case.
 
Shall we do this in a separate PR with a more illustrative example?

Reply:
@jonathancaevans 
Thanks for explaining your use case, would you mind opening another issue for this, or even better if you can open a PR to make this change.